trunk/projects/h5fs_test/test_readimage.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
```

Comment:
'./unsorted.hdf5' => sys.argv[1] could turn this into a more generic script; I intended it as a unit test for the readImage function (sorry about the hardcoding)